upgrade.sh

```diff
@@ -401,13 +403,17 @@ function renameNamespace()
     find ${testsDir} -name "*.php" -exec sed -i "" -e "s|${oldNamespace}|${newNamespace}|g" {} \;
     find ${testsDir} -name "*.php" -exec sed -i "" -e "s|/\*--- OLDNAMESPACE ---\*/|${oldNamespace}|g" {} \;
     find ${newNamespaceDir} -name "*.php" -exec sed -i "" -e "s|/\*--- OLDNAMESPACE ---\*/|${oldNamespace}|g" {} \;
+    find ${newNamespaceDir} -name "*.php" -exec sed -i "" -e "s|instanceof\ Stringable|instanceof\ \\\Illuminate\\\Support\\\Stringable|g" {} \;
 
-    echo "-- Reigning the renaming back in; bringing Carbon back to Illuminate/Support"
+    echo "-- Reigning the renaming back in; bringing Carbon, HtmlString, Str back to Illuminate/Support"
 
     # @todo: do this more cleanly
-    # Just in tests, fix namespaces for classes that are no longer provided in collections as of 8.0+
+    # Just in tests, fix namespaces for classes that are no longer provided in collections as of 8.0+ Illuminate\Support\Traits\Macroable, etc
     find ${testsDir} -name "*.php" -exec sed -i "" -e "s|Tightenco\\\Collect\\\Support\\\Carbon|Illuminate\\\Support\\\Carbon|g" {} \;
     find ${testsDir} -name "*.php" -exec sed -i "" -e "s|Tightenco\\\Collect\\\Support\\\HtmlString|Illuminate\\\Support\\\HtmlString|g" {} \;
+    find ${testsDir} -name "*.php" -exec sed -i "" -e "s|Tightenco\\\Collect\\\Support\\\Str|Illuminate\\\Support\\\Str|g" {} \;
+    find ${testsDir} -name "*.php" -exec sed -i "" -e "s|Illuminate\\\Support\\\Traits\\\Macroable|Tightenco\\\Collect\\\Support\\\Traits\\\Macroable|g" {} \;
+    find ${testSupportDirectory} -name "*.php" -exec sed -i "" -e "s|Illuminate\\\Support\\\Traits\\\Macroable|Tightenco\\\Collect\\\Support\\\Traits\\\Macroable|g" {} \;
```

Comment:
Why are these two necessary?

Reply:
`Macroable` is now required by the `Str` class. The `Str` class is under the `Illuminate\Support` namespace, so the `Macroable` also needs to be.  Since these files are not included in `tightenco/collect`, keeping them under `Illuminate\Support` makes sense.

Reply:
Right, but in this spot, you're renaming them from `Illuminate\Support\Traits\Macroable` to `Tightenco\Collect\Support\Traits\Macroable`